app/views/todos/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+  <div class="page-header">
+    <h1>Edit Todo</h1>
+  </div>
+</div>
+
+<div class='container'>
+    <%= render 'form', todo: @todo %>
```

Comment:
use 2 space indentation here as well.
Every file in your codebase should follow same indentation pattern.
In our codebase we use 2space